tests/unit/test_protocol_errors.py

```diff
@@ -22,6 +22,7 @@ class TestFrontendErrors(unittest.TestCase):
     """
     @classmethod
     def setUpClass(cls):
+        reload(frontend)
```

Comment:
I see.  Instead of reloading the entire module, would it be possible to just reset the flask app somehow, or reassign the frontend.app variable to a new instance?  That would seem more lightweight than reloading the entire module.  (It might make little difference now, but I could foresee a period down the line where either more bootstrap code is loaded into frontend that we don't want loaded again and/or frontend is split into several modules.)